### DISCUSSION TITLE
Combine discounts at checkout

### DISCUSSION THREAD

**Discussion by malo-malo** [Shopify/storefront-api-feedback#214] [Ideas & Requests]
Hello,

We are using the Storefront API to create carts, with multiple discounts (using the [discountCodes](https://shopify.dev/docs/api/storefront/2023-01/mutations/cartCreate#field-cartinput-discountcodes) attribute). The cart is correctly created, but when going to the checkout page for the order, only one discount is present.

I understand that discounts can be combined in a single order, but not when using the Storefront API
> Discounts can only be combined on orders that are made through the Online Store.

It would be very helpful if the discounts used when creating the cart were all present at checkout, will this be possible anytime soon ?

Thanks !

**Comment by johncraigcole** [comment]
Hey @malo-malo 👋 . Thanks for reporting this issue. 

You're correct. Assuming the discounts are combinable, both discounts specified with the Storefront API cart mutation `cartDiscountCodesUpdate` should flow through into checkout. I'll be sure to follow up here when the issue is resolved.

**Reply by johncraigcole** [reply]
Issue is now resolved.

**Reply by arobbins** [reply]
Thanks for the update

**Reply by malo-malo** [reply]
Thank you for the quick response !
